assignments/objects.js

```diff
@@ -18,27 +18,64 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
```

Comment:
```suggestion
speak: function() {
    return `Hello, my name is ${this.name}!`;
  }
}
```
The method could be declared here, rather than below. The way you have it is fine, it works, but this is easier for the next developer to have everything all in one place. Same with Antoinetta.